.gitattributes

```diff
@@ -0,0 +1,4 @@
+* text=auto eol=lf
+
+*.bat text eol=crlf
+*.jar binary
```

Comment:
Why are you adding this?

Reply:
The reason I listed [here](https://github.com/square/okhttp/pull/6773).